### PR TITLE
blog details

### PR DESCRIPTION
**Pull request by prank15062002**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Comment by prank15062002** [comment]
please ell me when the file will be merged so that i can submit my work

File: _data/blogs.yml [hunk 1]
```diff
@@ -207,3 +207,9 @@
   title: Interviewing a Teacher 
   img: Valley.jpg
   author_name: Yash Kothari
+  
+ - url: https://openblogsource.wordpress.com/2017/12/28/my-journey-open-source
```
Comment (nikit19) [inline _data/blogs.yml:211]: Please remove the space before `-`

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make the changes and upload the image.
You can close any one of the pull requests and work on the other .

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@prank15062002  Please avoid creating multiple PRs of the same issue and Please go through `README.md` on how to add add a blog 👍

File: _data/blogs.yml [hunk 2]
```diff
@@ -207,3 +207,9 @@
   title: Interviewing a Teacher 
   img: Valley.jpg
   author_name: Yash Kothari
+
```
Comment (0x48piraj) [inline _data/blogs.yml:210]: remove extra space.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts

**Review by ParthS007 — COMMENTED** [review]
@prank15062002  closing this PR as there are lot conflicts. Please read `README.md` on how to add a blog and ask on Gitter channel if you face any difficulties.

**Pull request closed without merging**